<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(218, 39, 120)</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:green;
font:20px &quot;Lucida Fax&quot;;</string>
    </property>
    <property name="text">
     <string> Random Number Generator</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>20</width>
      <height>541</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>431</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">::tab::selected{
background-color:rgb(87, 159, 170);
font:30px &quot;Lucida Fax&quot;;
width:200px;
height:40px;
}
::tab{
background-color:rgb(143, 255, 252);
font:30px &quot;Lucida Fax&quot;;
width:200px;
height:40px;
}
::pane{
color: rgb(85, 255, 0);
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>Max:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>Min:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;</string>
      </property>
      <property name="text">
       <string>48</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>Step:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>Avoid From:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>Avoid To:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>431</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>Avoid All Previous Numbers</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>Avoid These:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>431</width>
        <height>131</height>
       </rect>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <item>
       <property name="text">
        <string>Edit...</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>240</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
background-color:cyan;
border-radius:0px;
border:2px solid green;</string>
      </property>
      <property name="text">
       <string>A&amp;dd</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
background-color:cyan;
border-radius:0px;
border:2px solid green;</string>
      </property>
      <property name="text">
       <string>Dele&amp;te</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Sequence</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>Use Accompanied Filters</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
background-color:cyan;
border-radius:0px;
border:2px solid green;</string>
      </property>
      <property name="text">
       <string>Sa&amp;ve</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
background-color:cyan;
border-radius:0px;
border:2px solid green;</string>
      </property>
      <property name="text">
       <string>&amp;Discard</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>431</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
color: rgb(0, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>Sequence:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:20px &quot;Lucida Fax&quot;;
color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>Previous Numbers:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:15px &quot;Lucida Fax&quot;;
color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>Total:0 Numbers</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px &quot;Lucida Fax&quot;;
background-color:cyan;
border-radius:0px;
border:2px solid green;</string>
    </property>
    <property name="text">
     <string>&amp;Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px &quot;Lucida Fax&quot;;
background-color:cyan;
border-radius:0px;
border:2px solid green;</string>
    </property>
    <property name="text">
     <string>Sto&amp;p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px &quot;Lucida Fax&quot;;
background-color:cyan;
border-radius:0px;
border:2px solid green;</string>
    </property>
    <property name="text">
     <string>Spea&amp;k</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px &quot;Lucida Fax&quot;;
background-color:cyan;
border-radius:0px;
border:2px solid green;</string>
    </property>
    <property name="text">
     <string>&amp;Clear All</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>585</y>
      <width>741</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:green;
font:15px &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>All Settled</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>291</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLCDNumber" name="lcdNumber_3"/>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="lcdNumber_2"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font:30px &quot;Lucida Fax&quot;;
color:rgb(255, 170, 0);
background-color:rgb(199, 77, 171)</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;PREV&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">font:30px &quot;Lucida Fax&quot;;
color:rgb(255, 170, 0);
background-color:rgb(199, 77, 171)</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;TIMES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
